content/build/hydrobuild.md

```diff
@@ -401,7 +420,7 @@ If you want to stop using Hydrobuild, and remove it from your system, remove
 the builder using the `docker buildx rm` command.
 
 ```console
-$ docker buildx rm hydrobuild
+$ docker buildx rm cloud-<org>-default
```

Comment:
Right now we only have `default` builder's name, but as soon as an account will have multiple builders, having `default` here won't be be true anymore. That said, I guess it's ok to keep it for the time being.